Client/src/main/java/org/projectcardboard/client/controller/GameController.java

```diff
@@ -65,7 +68,8 @@ public void setCurrentGame(GameCopyMessage gameCopyMessage) {
             "battlemap6_middleground@2x");
         battleScene.show();
       } catch (Exception e) {
-        e.printStackTrace();
+        logger.warn("Failed to create BattleScene");
```

Comment:
Could be a bit more info here that isn't the stack trace. Maybe what playernumber it failed for, or some info from the gamecopymessage that would help identify what failed without logging the whole error.